core/src/common.cpp

```diff
@@ -409,7 +409,8 @@ namespace irods::http
 			const auto oidc_config_iter =
 				config.find(nlohmann::json::json_pointer{"/http_server/authentication/openid_connect"});
 			if (oidc_config_iter == std::end(config)) {
-				logging::debug("{}: No openid_connect configuration found in server configuration.", __func__);
+				logging::debug("{}: No 'openid_connect' stanza found in server configuration.", __func__);
+				logging::error("{}: Could not find bearer token matching [{}].", __func__, bearer_token);
```

Comment:
Is this new error message intentional?

Reply:
Yes. As a result of the oidc config not being defined, the bearer token cannot be found/matched.

Reply:
Oh, I see. Is there any value in keeping the `debug` message separate, then? Seems like having both in the same message would be just as well. But I'm not knowledgeable in this area, so it may be useful to keep them separated.

Reply:
Either way would be fine. I'm going to leave it as is and see how things turn out.

We can always adjust them later.